### PR TITLE
NMS-15274: Move severities chart

### PR DESCRIPTION
**Pull request by emarsh1995**
- Update `operation/deep-dive/alarms/alarm-advanced-search.adoc`
- Update `operation/deep-dive/bsm/business-service-map.adoc`, `/operational-status.adoc`
- Update `operation/deep-dive/events/event-advanced-search.adoc`, `/event-configuration.adoc`
- Update `reference/service-assurance/monitors/VmwareMonitor.adoc`

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here, or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready. Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://opennms.atlassian.net/browse/NMS-15274

### CODE REVIEW

File: docs/modules/operation/pages/deep-dive/bsm/operational-status.adoc [hunk 1]
```diff
@@ -2,50 +2,11 @@
...
-| Indeterminate
-| No severity could be associated with this event.
-| 1
-|===
+The overall status is calculated from the edges, using the map and reduce functions, and uses the <<deep-dive/events/event-configuration.adoc#severities, severities>> from associated events and alarms.
 
 If a business service changes its operational status, a `uei.opennms.org/bsm/serviceOperationalStatusChanged` event is generated.
-In cases where the operational status changed from Normal to a higher severity, a `uei.opennms.org/bsm/serviceProblem` event is generated and is assigned the same severity as the business service.
-When the business service returns to Normal severity, a `uei.opennms.org/bsm/serviceProblemResolved` event is generated.
+In cases where the operational status changed from `Normal` to a higher severity, a `uei.opennms.org/bsm/serviceProblem` event is generated and is assigned the same severity as the business service.
```
Comment (mmahacek) [inline docs/modules/operation/pages/deep-dive/bsm/operational-status.adoc:8]:
```suggestion
In cases where the operational status changes severity to a state other than `Normal`, a `uei.opennms.org/bsm/serviceProblem` event is generated and is assigned the same severity as the business service.
```

File: docs/modules/operation/pages/deep-dive/bsm/operational-status.adoc [hunk 2]
```diff
@@ -2,50 +2,11 @@
...
+When the business service returns to `Normal` severity, a `uei.opennms.org/bsm/serviceProblemResolved` event is generated.
 
 NOTE: Use the `serviceProblem` and `serviceProblemResolved` events for notifications or ticketing integration.
```
Comment (mmahacek) [inline docs/modules/operation/pages/deep-dive/bsm/operational-status.adoc:11]:
```suggestion
TIP: You can use the `serviceProblem` and `serviceProblemResolved` events for notifications or ticketing integration.
```

File: docs/modules/operation/pages/deep-dive/events/event-configuration.adoc [hunk 3]
```diff
@@ -91,18 +149,16 @@ The ordering of events inside each tributary file is also preserved.
...
 
-==== Reloading the event configuration
+=== Reload event configuration
 
 After making manual changes to `$\{OPENNMS_HOME}/etc/eventconf.xml` or any of its tributary files, you must restart the eventd daemon.
 You can trigger a reload of the daemon in the Karaf shell or by issuing the following command on the {page-component-title} server:
 
 [source, console]
-----
 $\{OPENNMS_HOME}/bin/send-event.pl uei.opennms.org/internal/reloadDaemonConfig -p 'daemonName Eventd'
```
Comment (mmahacek) [inline docs/modules/operation/pages/deep-dive/events/event-configuration.adoc:164]: Don't need the `\` in a `[source]` block as it renders there.
```suggestion
$OPENNMS_HOME}/bin/send-event.pl uei.opennms.org/internal/reloadDaemonConfig -p 'daemonName Eventd'
```

File: docs/modules/reference/pages/service-assurance/monitors/VmwareMonitor.adoc [hunk 4]
```diff
@@ -1,60 +1,66 @@
...
+This monitor is part of the VMWare integration that is provided by provisiond.
+It tests the power state of a virtual machine or host system.
+If the power state is `poweredOn`, the service is considered to be up; if it is `poweredOff`, the service is marked as down.
+By default, a power state of `standBy` will also mark the service as down.
+
+If you use https://www.vmware.com/content/dam/digitalmarketing/vmware/en/pdf/techpaper/Distributed-Power-Management-vSphere.pdf[Distributed Power Management] (automatic PDF download), you can change the `standBy` state (see the `ignoreStandBy` parameter).
+
+Optionally, a service down event can be triggered if there are pending alarms in vCenter that have not been acknowledged or cleared.
+
+== Import hosts
```
Comment (mmahacek) [inline docs/modules/reference/pages/service-assurance/monitors/VmwareMonitor.adoc:13]: Could reference the `reference:provisioning/handlers/vmware.adoc` section that's dedicated to setting up the VMware import process.  Also, that section needs some updating for the v30+ External Requistions UI.

**Review by mmahacek — COMMENTED** [review]
I like the Advanced Search page updates to use tables.

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Looks great. Just a few minor changes in addition to Mark's.

File: docs/modules/reference/pages/service-assurance/monitors/VmwareMonitor.adoc [hunk 5]
```diff
@@ -1,60 +1,66 @@
...
+
+The credentials for accessing the status of a VMware entity are collected from the appropriate vCenter instance using settings from `$\{OPENNMS_HOME}/etc/vmware-config.xml`.
+This file also sets specific asset fields to be assigned to imported virtual machines and host systems.
+
+The VMware integration populates the following asset fields via provisiond:
 
 * VMware Management Server
 * VMware Managed Entity Type
-* foreignId (which contains an internal VMware vCenter Identifier)
+* `foreignId` (which contains an internal VMware vCenter Identifier)
```
Comment (emarsh1995) [inline docs/modules/reference/pages/service-assurance/monitors/VmwareMonitor.adoc:25]:
```suggestion
* `foreignId` (which contains an internal VMware vCenter Identifier)

For more information, see xref:reference:/provisioning/handlers/vmware.adoc[].
```

**Review by Bonrob2 — COMMENTED** [review]
LGTM! Thank you!

**Review by mmahacek — APPROVED** [review]

**Pull request merged**